Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1809,11 +1811,12 @@ namespace Remotion.BocAutoCompleteReferenceValue
             maxHeight = '';
         }
 
-        const availableWidth = position.left + LayoutUtility.GetOuterWidth(reference);
-        const minWidth = LayoutUtility.GetOuterWidth(reference);
-        const maxWidth = Math.min (isNaN (options.maxWidth) ? LayoutUtility.GetOuterWidth(reference) : options.maxWidth, availableWidth);
-        const requiredWidth = contentWidth + 30;
-        const elementWidth = Math.max (Math.min (requiredWidth, maxWidth), minWidth);
+        const referenceWidth = LayoutUtility.GetOuterWidth(reference);
```

Comment:
Something's still fishy:
![image](https://user-images.githubusercontent.com/6972548/206115704-8c86c588-a7ff-4421-878f-a4008897622b.png)

It does work when the text is generally shorter than the input field
![image](https://user-images.githubusercontent.com/6972548/206116154-f2568fe6-9297-4976-9147-c623b7aedd6d.png)

Maybe the padding isn't correct?